src/indent.cpp

```diff
@@ -493,7 +493,8 @@ static Chunk *oc_msg_block_indent(Chunk *pc, bool from_brace,
    LOG_FUNC_ENTRY();
    Chunk *tmp = Chunk::NullChunkPtr;
 
-   if (pc != nullptr)
+   if (  pc != nullptr
+      && pc->IsNotNullChunk())
```

Comment:
```if (pc != nullptr)``` was added to prevent SEGV during the transition process of the code improvement. At the end this will simply be ```tmp = pc->GetPrevNc();``` without any ```if```, because Chunk pointers won't be null.
Checking for ```pc->IsNotNullChunk()``` is not wrong, but it is not necessary either. If ```pc``` is a null chunk, ```pc->GetPrevNc()``` will return a Chunk::NullChunkPtr, with no effect on the current value of ```tmp```

Reply:
We can leave the original as is, as explained in previous comment.